backend/sources/chat-plugin/src/main/java/co/airy/core/chat_plugin/ChatController.java

```diff
@@ -29,11 +29,13 @@
     private final ObjectMapper objectMapper;
     private final Stores stores;
     private final Jwt jwt;
+    private final Mapper mapper;
 
-    public ChatController(Stores stores, Jwt jwt, ObjectMapper objectMapper) {
+    public ChatController(Stores stores, Jwt jwt, ObjectMapper objectMapper, Mapper mapper) {
```

Comment:
Chat plugin was not using the message mapping yet. So I am adding it to this PR.